test/recurly/BaseClient.test.js

```diff
@@ -1,28 +1,30 @@
-/* globals describe, it */
+/* globals describe, it, beforeEach, afterEach */
 
 require('../test_helper')
+const sinon = require('sinon')
+const { MyResource } = require('../mock_resources')
 const assert = require('assert').strict
+const recurly = require('../../lib/recurly')
 const pkg = require('../../package.json')
-const BaseClient = require('../../lib/recurly/BaseClient')
+const MyClient = require('../mock_client')
 
-class TestClient extends BaseClient {
-  apiVersion () {
-    return 'v2020-01-01'
-  }
-}
-
-const client = new TestClient('myapikey')
+const client = new MyClient('myapikey')
```

Comment:
It seems like for safety this should be created `beforeEach` instead of restored, in case the MyClient has some behavioral aspect that aren't covered by its sandbox.

Reply:
good point